docs/source/configuration/apollo-telemetry.mdx

```diff
@@ -17,7 +17,7 @@ More information on usage reporting is available in the [Studio documentation](/
 
 ## Enabling field-level instrumentation
 
-Field-level instrumentation (also known as FTV1 tracing) is disabled by default. To enable it, add the following to your router configuration:
+Field-level instrumentation (also known as FTV1 tracing) is set to `0.01` by default. To enable it, add the following to your router configuration:
```

Comment:
I feel like this should be telling you how to disable it now, not how to enable it (since it is already enabled).